### ISSUE TITLE
Calculate the multivariate Gaussian prior logpdf in log space

### ISSUE THREAD

**Issue by rcw5890** [pints-team/pints#1042]
I was getting some overflow errors which traced back to the evaluation of the multivariate Gaussian prior, which looks like it takes the log after evaluating the straight pdf.

Let me know if there was a good reason for this as opposed to calling `scipy.stats.multivariate_normal.logpdf` directly, which helps avoid numerical issues.

```
def __call__(self, x):
    return np.log(
        scipy.stats.multivariate_normal.pdf(
            x, mean=self._mean, cov=self._cov))
```
https://github.com/pints-team/pints/blob/master/pints/_log_priors.py

**Comment by ben18785** [comment]
Yep, there's no good reason for this. Can you create a ticket for it and
perhaps a PR if it's not too much of a change? Thanks, Ben

**Comment by MichaelClerx** [comment]
Yeah feel free to do a PR and tag me as a reviewer for that @rcw5890 !

**Comment by MichaelClerx** [comment]
Nice catch!

**Comment by rcw5890** [comment]
Should be fixed as of #1043
